src/get_class_instances.ts

```diff
@@ -0,0 +1,39 @@
+function class_exists(class_name: string) {
+    return !!Java.classFactory.loader?.find(class_name);
+}
+
+function get_class_object(class_name: string) {
+    const classes = Java.enumerateLoadedClassesSync();
+
+    for (const class_object of classes) {
+        if (class_object.match(class_name)) {
+            return Java.use(class_object);
+        }
+    }
+
+    throw "Error can't find class " + class_name;
```

Comment:
Aren't there any cool exception-types in typescript?
Just like those ValueError, OSError, ... from python..? 🥲🥲